openapi/tests/gh-5792/src/test/java/io/helidon/openapi/tests/yamlparsing/SnakeYAMLV1Test.java

```diff
@@ -39,10 +41,16 @@ class SnakeYAMLV1Test {
         this.client = client;
     }
 
+    @SetUpServer
+    static void server(WebServerConfig.Builder server) {
+        server.addFeature(OpenApiFeature.builder()
+                                  .staticFile("target/test-classes/petstore.yaml")
+                                  .permitAll(true)
+                                  .build());
+    }
     @SetUpRoute
     static void routing(HttpRouting.Builder routing) {
-        routing.addFeature(OpenApiFeature.builder()
-                                   .staticFile("target/test-classes/petstore.yaml"));
+        routing.get("/greet", (req, res) -> res.send("Hi!"));
```

Comment:
The `routing` method is not needed any more right ? that `/greet` endpoint seems to be unused.

Reply:
this is intentionally done so you cannot access the endpoint even if you do not have security configured. To protect people who forget they are exposing an endpoint with data that could be missused.

So you have two options:
1.  configure real security and protect the endpoint
2. explicitly mark the endpoint to `permit-all`, so you make an action of affirmation of this problem
3.

Reply:
Current implementation of JUnit extensions requires the set up route method to create an instance of the client with correct routing (including server features).
To change this behavior, we would need to refactor that approach. I do not think I can do it quick enough.
Would it be better to have the method empty in the test class?

Reply:
I'd rather have it empty.

Reply:
Fixed